apkleaks/apkleaks.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, args):
 		self.output = tempfile.mkstemp(suffix=".%s" % ("json" if self.json else "txt"), prefix=self.prefix)[1] if args.output is None else args.output
 		self.fileout = open(self.output, "%s" % ("w" if self.json else "a"))
 		self.pattern = os.path.join(str(Path(self.main_dir).parent), "config", "regexes.json") if args.pattern is None else args.pattern
-		self.jadx = find_executable("jadx") if find_executable("jadx") is not None else os.path.join(str(Path(self.main_dir).parent), "jadx", "bin", "jadx%s" % (".bat" if os.name == "nt" else ""))
+		self.jadx = find_executable("jadx") if find_executable("jadx") is not None else os.path.join(str(Path(self.main_dir).parent), "jadx", "bin", "jadx%s" % (".bat" if os.name == "nt" else "")).replace("\\","/")
```

Comment:
Is it necessary? I mean, you've replaced the `comm` variable afterward.

Reply:
No, my point is: instead of using `replace` twice, why not just concatenate it on one of the lines?

Reply:
yes, it need twice `replace`. line 36 replace \ in jadx path, so jadx path won't be quoted, line 98 replace ' in other file path, so it could be correctly parse on windows.